css/main.css

```diff
@@ -0,0 +1,18 @@
+section {
+	margin-left: 33%;
+	margin-right: 33%;
+}
+
+input{
+	width: 100%
+}
+button{
+
+	width: 25%;
+}
+
+@media screen and (max-width: 700px) {
```

Comment:
You'll need to include your section inside of here -- it should expand to take up more size on a smaller view.